### ISSUE TITLE
Typescript plugins don't seem to transpile using the start command

### ISSUE THREAD

**Issue by Mike-Dax** [PepsRyuu/nollup#8]
I can't seem to get typescript to work. I've tried both these plugins:

https://github.com/rollup/rollup-plugin-typescript
https://github.com/ezolenko/rollup-plugin-typescript2/

Neither appear to transpile the code.

```
SyntaxError: Unexpected token (6:5)
    type thing = number
        ^
Trace: Error trace
    at bundle (/Users/michaelorenstein/Projects/nollup/lib/index.js:496:17)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:279:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:696:3)
```

Building them manually with my usual config, and with the build script provided both work. 

It's just the 'start' command that seems to have issues.

I've branched off as far as I got:

https://github.com/Mike-Dax/nollup/tree/typescript-example/examples/typescript-react

Thank you for your help.

**Comment by PepsRyuu** [comment]
I pushed a fix for this to the master branch. https://github.com/PepsRyuu/nollup/commit/9186f7a9b1e8a909cd64dba1f469fc975ba97b83

Apparently, the "filter" function that plugins use to determine whether or not to parse the file, ignores relative paths. Converting the relative path to an absolute path seems to resolve the issue. 

Let me know if this works for you! I'll make a release once confirmed.

**Comment by Mike-Dax** [comment]
It works! Thank you.

**Comment by PepsRyuu** [comment]
Released in 0.2.1. :)
